crates/modelardb_compression/src/models/swing.rs

```diff
@@ -353,29 +350,70 @@ mod tests {
     // Tests for Swing.
     proptest! {
     #[test]
-    fn test_can_fit_sequence_of_finite_value_with_error_bound_zero(value in ProptestValue::ANY) {
-        can_fit_sequence_of_value_with_error_bound_zero(value)
+    fn test_can_fit_sequence_of_finite_value_with_absolute_error_bound(value in ProptestValue::ANY) {
```

Comment:
In the other tests where `ERROR_BOUND_ZERO` is used the test is named `with_absolute_error_bound_zero` or `with_relative_error_bound_zero`. Maybe be consistent with this naming scheme throughout the tests.

Reply:
Fixed.